modules/server/src/topics/documentationtopic.cpp

```diff
@@ -47,19 +46,19 @@ void DocumentationTopic::handleJson(const nlohmann::json& json) {
     // Do not parse generated json. Instead implement ability to get
     // ghoul::Dictionary objects from ScriptEngine, FactoryManager, and KeybindingManager.
     if (requestedType == "lua") {
-        response = global::scriptEngine->generateJson();
+        response = DocEng.generateScriptEngineJson();
     }
     else if (requestedType == "factories") {
-        response = FactoryManager::ref().generateJson();
+        response = DocEng.generateFactoryManagerJson();
     }
     else if (requestedType == "keyboard") {
-        response = global::keybindingManager->generateJson();
+        response = DocEng.generateKeybindingsJson();
     }
     else if (requestedType == "asset") {
-        response = global::keybindingManager->generateJson();
+        // TODO: Add asset documentation here
```

Comment:
What should be here? Why does this type exist and is it still needed? (I do see that it used to contain the same information as the keyboard type)

If it is, and if this comment should stay around, we try to attach a date and some reference to the person who wrote it, as well as start with an @ sign. Like this: 
```
@TODO (ylvse, 2023-03-05) Add asset documentation here
```
That helps the next person who comes across the comment to figure out whether it is still relevant :)

Reply:
So I think @micahnyc originally created this topic. I just wanted to preserve what was already there, hence my comment, but I actually don't really know what @micahnyc originally planned.